packages/actor-init-sparql/components/Actor/Init/Sparql.jsonld

```diff
@@ -1,6 +1,7 @@
 {
   "@context": [
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-init-sparql/^1.0.0/components/context.jsonld",
```

Comment:
Are all these 1.0.0 references correct?

Reply:
Yes, _all_ context use `^1.0.0` at the moment. This should only be updated on major releases.

Reply:
There are also some non-context references to 1.0.0, e.g.: https://github.com/comunica/comunica/pull/208/files#diff-1fccd41cbc783b6aea9b344380c5a8bfR116 (might also be some others)